packages/website/src/components/layout/EditorTabs.module.css

```
@@ -12,12 +12,25 @@
   --ifm-button-border-color: var(--ifm-color-emphasis-200);
 
   font-size: 0.75rem;
+  display: flex;
 }
 
 .tabStyle svg {
   margin-left: 0.3rem;
 }
 
+.tabStyle div {
```

Selectors that target all tags of a certain type can accidentally target nested children. If we were to, say, add a child component that happens to render `<div>`s, this would target that child's elements accidentally. Let's use a specific class name:

```suggestion
.tabErrors {
```

---

i'm agree, so i change it to suggestion code